File: 3038. Maximum Number of Operations With the Same Score I.php
class Solution {
    function maxOperations($nums) {
    $score = $nums[0] + $nums[1];
    $count = 1;
    for ($i = 2; $i <= count($nums) - 2 && $nums[$i] + $nums[$i + 1] == $score; $i += 2) {$count++;}
    return $count;
    }
}

File: 3038. Maximum Number of Operations With the Same Score I.rs
impl Solution {
    pub fn max_operations(nums: Vec<i32>) -> i32 {
    let mut score: i32 = nums[0] + nums[1];
    let mut count: i32 = 1;
    for i in (2..=nums.len()-2).step_by(2) {
        if nums[i] + nums[i+1] == score {
            count += 1;
        } else {break;}
    }
    count 
    }
}
